docs/help.rst

```diff
@@ -70,68 +75,95 @@ contact support@robusta.dev or go to `our slack channel <https://bit.ly/robusta-
 
-    .. details:: Discovery Error
+        If your cluster has 20 Nodes or less, set Prometheus memory request to TBD
 
-        .. code-block::
+        If using a test cluster like Kind/Colima, re-install Robusta with the ``isSmallCluster=true`` property:
 
-            2023-04-17 23:37:43.019 ERROR    Discovery process internal error
-            2023-04-17 23:37:43.022 INFO     Initialized new discovery pool
-            2023-04-17 23:37:43.022 ERROR    Failed to run publish discovery for robusta_ui_sink
-            Traceback (most recent call last):
-              File "/app/src/robusta/core/sinks/robusta/robusta_sink.py", line 175, in __discover_resources
-                results: DiscoveryResults = Discovery.discover_resources()
-              File "/app/src/robusta/core/discovery/discovery.py", line 288, in discover_resources
-                raise e
-              File "/app/src/robusta/core/discovery/discovery.py", line 280, in discover_resources
-                return future.result()
-              File "/usr/local/lib/python3.9/concurrent/futures/_base.py", line 446, in result
-                return self.__get_result()
-              File "/usr/local/lib/python3.9/concurrent/futures/_base.py", line 391, in __get_result
-                raise self._exception
-            concurrent.futures.process.BrokenProcessPool: A process in the process pool was terminated abruptly while the future was running or pending.
+        .. code-block:: bash
 
-.. details:: AlertManager Silences are Disappearing
+                helm install robusta robusta/robusta -f ./generated_values.yaml --set clusterName=<YOUR_CLUSTER_NAME> --set isSmallCluster=true
 
-        This happens when AlertManager does not have persistent storage enabled.
 
-        When using Robusta's embedded Prometheus Stack, persistent storage is enabled by default.
+.. details:: robusta-runner isn't working or has exceptions
 
-        For other Prometheus distributions set the following Helm value (or it's equivalent):
+        Start by checking the logs for errors:
 
-        .. code-block::
+        .. code-block:: bash
 
-                  # this is the setting in in kube-prometheus-stack
-                  # the exact setting will differ for other Prometheus distributions
-                  alertmanager:
-                    alertmanagerSpec:
-                      storage:
-                        volumeClaimTemplate:
-                          spec:
-                            accessModes: ["ReadWriteOnce"]
-                            resources:
-                              requests:
-                                storage: 10Gi
+                kubectl get pods -A | grep robusta-runner # get the name and the namespace of the robusta pod
+                kubectl logs -n <NAMESPACE> <ROBUSTA-RUNNER-POD-NAME> # get the logs
+
+        Check the pod's memory consumption. If necessary, increase the memory request in the Helm values:
```

Comment:
I think the connection between the memory issue and the exception is not very clear
I think I would put this in the discovery error section.

- first the stack trace
- This means you need to increase runner memory
- This is how to do it

Similar to what you did on the holmes exception below